src/Apps/Order/Components/ExpressCheckout/index.tsx

```diff
@@ -29,9 +29,16 @@ type Seller = Exclude<
 
 export const ExpressCheckout = ({ order, setShowSpinner }: Props) => {
   const orderData = useFragment(ORDER_FRAGMENT, order)
+  const [isChrome, setIsChrome] = useState<boolean | null>(null)
+
+  useEffect(() => {
+    const chrome =
+      typeof navigator !== "undefined" && /Chrome/.test(navigator.userAgent)
+    setIsChrome(chrome)
+  }, [])
```

Comment:
If this is a variable we set one time, I don't think it needs the `useEffect`/`useState` - can just be
```
 const isChrome =  !!(typeof navigator !== "undefined" && /Chrome/.test(navigator.userAgent))
```

Or if we want to memoize the value we can do 
```
const isChrome = useMemo(() => {
    return !!(typeof navigator !== "undefined" && /Chrome/.test(navigator.userAgent))
  }, [])
```

Reply:
All good, makes sense now! Thanks.